zpages/zpages.go

```diff
@@ -31,16 +31,40 @@
 package zpages // import "go.opencensus.io/zpages"
 
 import (
+	"fmt"
+	"log"
+	"net"
 	"net/http"
 )
 
-// Handler is an http.Handler that serves the zpages.
+// Handler is deprecated: Use NewHandler.
```

Comment:
Once this is removed according to the deprecation policy, we can move the init logic into NewHandler.